### PR TITLE
Added support for transform-decorators-legacy

### PR DESCRIPTION
**Pull request by mweststrate**
To be able to actually use decorators :) (Standard transforms just throws an error about the specs not being finalized).

Thanks for this nifty project btw!

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ export const availablePlugins = {
   'transform-class-constructor-call': require('babel-plugin-transform-class-constructor-call'),
   'transform-class-properties': require('babel-plugin-transform-class-properties'),
   'transform-decorators': require('babel-plugin-transform-decorators'),
+  'transform-decorators-legacy': require('babel-plugin-transform-decorators-legacy').default, // <- No clue. Nope.
```
Comment (Daniel15) [inline src/index.js]: LOL, what's up with the `.default`?

**Comment by Daniel15** [comment]
Apparently there's merge conflicts (not sure how)... Could you please rebase this? I can merge it once rebased :)

**Comment by mweststrate** [comment]
Conflicts were a result of the dependencies update in package.json. Fixed that. The `default` issue is probably caused by a different ES6 module format being used in legacy decorators. It works correctly though, but maybe this deviation should be normalized by the plugin normalizer (if there is a `.default` exports, prefer that).

**Comment by Daniel15** [comment]
> The default issue is probably caused by a different ES6 module format being used in legacy decorators

If it works, it's fine. ¯\_(ツ)_/¯

**Comment by Daniel15** [comment]
> The default issue is probably caused by a different ES6 module format being used in legacy decorators

If it works, it's fine. ¯\\\_(ツ)\_/¯

**Pull request merged**